bauhaus/workflows/secondary/hqMetrics.py

```diff
@@ -0,0 +1,81 @@
+from bauhaus import Workflow
+from .datasetOps import *
+from .mapping import genChunkedMapping
+from bauhaus.experiment import (InputType, ResequencingConditionTable)
+
+__all__ = ["HQMetricsWorkflow"]
+
+class HQMetricsWorkflow(Workflow):
+    @staticmethod
+    def name():
+        return "HQMetrics"
+
+    @staticmethod
+    def conditionTableType():
+        return ResequencingConditionTable
+
+    def generate(self, pflow, ct):
+        outputDict = {}
+        pflow.bundleScript("R/hqrf_pbbamr.R")
+        pflow.genRuleOnce("hqrf_pbbamr",
+                          "$grid Rscript --verbose R/hqrf_pbbamr.R $origbam xscraps $in $out")
+
+        pflow.genRuleOnce("bam2bam_nohq",
+                          "$grid bam2bam -j $ncpus -b $ncpus --fullHQ --adapters $adapters -o $outPrefix $in")
+
+        pflow.genRuleOnce("subreadset_create",
+                          "$grid dataset create $out $in")
+
+        for condition in ct.conditions:
+            with pflow.context("condition", condition):
+                reference = ct.reference(condition)
+                if ct.inputType == InputType.SubreadSet:
+                    inputs = ct.inputs(condition)
+
+                    if len(inputs) != 1 :
```

Comment:
In general this isn't true--the condition table spec allows multiple inputs to be included under a condition.  

Is your claim that this should not be allowed for this workflow?  If so, the right place to enforce it would actually be at the condition table validation.

Anyway I can live with merging this as is, I just want to understand your intention.